### PR TITLE
fix(Chat/CompactMessageView): removed space above image in msg

### LINKED ISSUE #6855: Gif/images
### Description

Gif and Images have this huge space at the top that needs to be adjusted

1. join a channel
2. send a gif

and

1. join a community
2. upload an image to a community channel

current

Screen Shot 2022-08-05 at 2 31 23 PM
Screen Shot 2022-08-05 at 2 30 35 PM

expected

Screen Shot 2022-08-05 at 2 30 23 PM

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #6855

### What does the PR do

Chat/CompactMessageView: removed space above image in msg

### Affected areas
Chat/CompactMessageView

### Screenshot of functionality (including design for comparison)
f

### CODE REVIEW

File: ui/imports/shared/views/chat/CompactMessageView.qml [hunk 1]
```diff
@@ -547,11 +548,7 @@ Item {
                 anchors.topMargin: isEmoji ? 2 : 0
                 anchors.left: parent.left
                 anchors.right: parent.right
-                // using a padding instead of a margin let's us select text more easily
-                anchors.leftMargin: -leftPadding
-                textField.leftPadding: leftPadding
```
Comment (alexandraB99) [inline ui/imports/shared/views/chat/CompactMessageView.qml:552]: @jrainville could you please check this whether is ok to remove, it was introduced in https://github.com/status-im/status-desktop/commit/bffe888b0db68a5442e2acd6121008cde636aee5

File: ui/imports/shared/views/chat/ChatTextView.qml [hunk 2]
```diff
@@ -24,7 +24,21 @@ Item {
 
     z: 51
 
-    implicitHeight: visible ? (showMoreLoader.active ? childrenRect.height - 10 : chatText.height) : 0
+    implicitHeight: {
+        if (visible) {
```
Comment (faust4exe) [inline ui/imports/shared/views/chat/ChatTextView.qml:28]: I wonder if using form like
```
if(!visible) return 0
...
```
is better ?

**Review by faust4exe — APPROVED** [review]

**Review by caybro — APPROVED** [review]

File: ui/imports/shared/views/chat/ChatTextView.qml (hunk 2 above)
Comment (faust4exe) [reply ui/imports/shared/views/chat/ChatTextView.qml:28]: @alexandraB99 Not sure if very good reason. But I remember some practices about adding in the beginning the conditions which lead to exit from function and another one was about nesting levels, which you should keep lower.

Comment (alexandraB99) [reply ui/imports/shared/views/chat/ChatTextView.qml]: updated!

File: ui/imports/shared/views/chat/ChatTextView.qml [hunk 3]
```diff
@@ -24,7 +24,21 @@ Item {
...
+            if (chatText.visible) {
+                if (root.veryLongChatText && !root.readMore) {
+                    return Math.min(chatText.contentHeight, 200);
+                } else {
+                    return chatText.getText(0, chatText.text.length) !== "" ? chatText.contentHeight : 0;
+                }
+            } else if (showMoreLoader.active ) {
```
[the remark this answers is not in the document]
Comment (alexandraB99) [reply ui/imports/shared/views/chat/ChatTextView.qml]: I guess it will fall in 0

**Review by faust4exe — APPROVED** [review]
Looks good!

**Review by borismelnik — APPROVED** [review]

**Pull request merged**